### PR TITLE
Add Gopi Vaibhav's profile

### LINKED ISSUE #4726: [Community] Member Profile: Gopi Vaibhav
**Current Behavior**

@gopivaibhav  has been a consistent contributor and community member.

**Desired Situation**
Let's recognize  @gopivaibhav  as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub:  @gopivaibhav
- Twitter: [gopi_vaibhav](https://twitter.com/gopi_vaibhav)
- LinkedIn: [gopivaibhav](https://www.linkedin.com/in/gopivaibhav/) 
- Meshery Cloud: [85339de7-8ca2-4543-8793-1ebb1e76bf99](https://meshery.layer5.io/user/85339de7-8ca2-4543-8793-1ebb1e76bf99)
- Link to profile picture:

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](https://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by Nithinyanamaddi**
**Description**

This PR fixes #4726

**Notes for Reviewers**
Added Gopi Vaibhav's Profile to the members page

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit c321e9e1b3ad3714ac81ae44f84349f1a7888e19 at: https://64d7ca5a763dcc246f250bc8--layer5.netlify.app

File: src/collections/members/gopi-vaibhav/index.mdx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+position: Contributor
+image_path: ./gopi-vaibhav.webp
+github: gopivaibhav
+twitter: gopi_vaibhav
+linkedin: gopivaibhav
+layer5: 85339de7-8ca2-4543-8793-1ebb1e76bf99
+location: Lucknow, India
+bio: "I am a Final year CSE student from IIIT Lucknow. I am an experienced MERN Stack Developer with a solid background in Python web frameworks, notably Django. My journey in the tech world has led me to actively contribute to prominent open-source projects like Layer5 and Oppia, through which I started learning to use Golang, Gatsby, GraphQL, Docker, k8s, etc. Aside from coding, I am a passionate student who is continuously pursuing new hobbies, such as trying new video games and participating in outdoor activities. I am constantly open to new and intriguing possibilities and look forward to meeting like-minded professionals. If you have a passion for web development, open-source contributions, or technology in general, let's connect and discuss how we might collaborate to create extraordinary digital solutions! 🚀"
+badges: 
+  - Community
```
Comment (gopivaibhav) [inline src/collections/members/gopi-vaibhav/index.mdx:12]: It should be in lowercase `community` as it is not showing the profile when community tag is applied as a filter

File: src/collections/members/gopi-vaibhav/index.mdx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+meshmate: no
```
Comment (gopivaibhav) [inline src/collections/members/gopi-vaibhav/index.mdx:13]: It is still giving MeshMate icon on opening the profile

**Comment by abhijeetgauravm** [comment]
hey @Nithinyanamaddi Please sign off your commits too as DCO check is failing!

**Comment by Nithinyanamaddi** [comment]
Hey @abhijeetgauravm I signed off my commits using `git commit -s -m "Message"` but when I clicked on Update branch in the GitHub, it Merged from the master branch without signing off. Can you help me how I can sign off when I commit from GitHub?

**Comment by l5io** [comment]
🚀 Preview for commit 0ef85176f65b7df78522cde3761b8112aae34884 at: https://64d916377b37a0356201a6d1--layer5.netlify.app

**Comment by abhijeetgauravm** [comment]
> Hey @abhijeetgauravm I signed off my commits using `git commit -s -m "Message"` but when I clicked on Update branch in the GitHub, it Merged from the master branch without signing off. Can you help me how I can sign off when I commit from GitHub?

hey @Nithinyanamaddi when u r going to make commits just do this in terminal
git config --global user.name "Your Name"
git config --global user.email "your.email@example.com"
git add .
git commit -s -m "your msg"
git push

It will work!

**Review by Chadha93 — APPROVED** [review]
Thanks everyone, @Nithinyanamaddi covering you on DCO

**Pull request merged**